nixpkgs_review/cli/__init__.py

```diff
@@ -121,7 +121,7 @@ def read_github_token() -> Optional[str]:
         try:
             with open(path) as f:
                 for line in f:
-                    token_match = re.match(r"\s*oauth_token:\s+([a-f0-9]+)", line)
+                    token_match = re.match(r"\s*oauth_token:\s+((?:gh[pousr]_)?[a-f0-9]+)", line)
```

Comment:
I think we only support ghp tokens and I think we should limit the filter to them.

Reply:
Fine with me, that'd make this change much simpler!  Will update soon unless anyone else chimes in :muscle: